roles/pulp3-content/templates/pulp_content_app.service.j2

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Pulp Content App
+After=network-online.target
+Wants=network-online.target
+
+# This service will break if left running while PostgreSQL restarts.
+BindsTo=postgresql.service
+After=postgresql.service
+
+[Service]
+EnvironmentFile=-/etc/default/pulp_workers
+EnvironmentFile=-/etc/default/pulp_workers_%i
```

Comment:
Should we be sharing environment files with the workers?

Reply:
I don't think so. That is supposed to source [this file's contents](https://github.com/pulp/pulp/blob/master/pulpcore/etc/default/pulp_workers) but that doesn't even make sense anymore for pulp3 anyway and should be removed (I think). Does this make sense to you?